packages/rpc/src/__tests__/HttpClient.test.ts

```diff
@@ -43,49 +43,56 @@ const ingressEgressEnvironment: z.input<typeof cfIngressEgressEnvironment> = {
     Polkadot: { DOT: '0x0' },
     Bitcoin: { BTC: '0x0' },
     Arbitrum: { ETH: '0x0', USDC: '0x0' },
-    // Solana: { SOL: '0x0' },
+    Solana: { SOL: '0x0' },
+    Assethub: { DOT: '0x0', USDC: '0x0', USDT: '0x0' },
   },
   ingress_fees: {
     Ethereum: { ETH: '0x55730', FLIP: '0x0', USDC: '0x0', USDT: '0x0' },
     Polkadot: { DOT: '0xbc28f20' },
     Bitcoin: { BTC: '0x4e' },
     Arbitrum: { ETH: '0x574b457d400', USDC: '0x231b' },
-    // Solana: { SOL: '0xb0' },
+    Solana: { SOL: '0xb0' },
+    Assethub: { DOT: '0x0', USDC: '0x0', USDT: '0x0' },
   },
   egress_fees: {
     Ethereum: { ETH: '0x77a10', FLIP: '0x0', USDC: '0x0', USDT: '0x0' },
     Polkadot: { DOT: '0xbc4d910' },
     Bitcoin: { BTC: '0xb0' },
     Arbitrum: { ETH: '0x74645ca7000', USDC: '0x2701' },
-    // Solana: { SOL: '0xb0' },
+    Solana: { SOL: '0xb0' },
+    Assethub: { DOT: '0x0', USDC: '0x0', USDT: '0x0' },
   },
   witness_safety_margins: {
     Bitcoin: 2,
     Polkadot: null,
     Ethereum: 2,
     Arbitrum: 1,
-    // Solana: 1,
+    Solana: 1,
+    Assethub: null,
   },
   egress_dust_limits: {
     Ethereum: { ETH: '0x1', FLIP: '0x1', USDC: '0x1', USDT: '0x1' },
     Polkadot: { DOT: '0x1' },
     Bitcoin: { BTC: '0x258' },
     Arbitrum: { ETH: '0x1', USDC: '0x1' },
-    // Solana: { SOL: '0x1' },
+    Solana: { SOL: '0x1' },
+    Assethub: { DOT: '0x0', USDC: '0x0', USDT: '0x0' },
```

Comment:
we will have USDT?

Reply:
yep https://linear.app/chainflip/issue/WEB-1772/add-assethub-chain-and-assets-to-sdk-and-do-a-private-release